Source/Core/VideoCommon/GraphicsModSystem/Runtime/Actions/PrintAction.cpp

```diff
@@ -5,27 +5,30 @@
 
 #include "Common/Logging/Log.h"
 
-void PrintAction::OnDrawStarted(bool*)
+void PrintAction::OnDrawStarted(const GraphicsModActionData::Input::DrawStarted&,
+                                GraphicsModActionData::Output::DrawStarted*)
 {
   INFO_LOG_FMT(VIDEO, "OnDrawStarted Called");
 }
 
-void PrintAction::OnEFB(bool*, u32 texture_width, u32 texture_height, u32* scaled_width,
-                        u32* scaled_height)
+void PrintAction::OnEFB(const GraphicsModActionData::Input::EFB& input,
+                        GraphicsModActionData::Output::EFB* output)
 {
-  if (!scaled_width || !scaled_height)
+  if (!output)
     return;
 
-  INFO_LOG_FMT(VIDEO, "OnEFB Called. Original [{}, {}], Scaled [{}, {}]", texture_width,
-               texture_height, *scaled_width, *scaled_height);
+  INFO_LOG_FMT(VIDEO, "OnEFB Called. Original [{}, {}], Scaled [{}, {}]", input.texture_width,
+               input.texture_height, output->scaled_width, output->scaled_height);
 }
 
-void PrintAction::OnProjection(Common::Matrix44*)
+void PrintAction::OnProjection(const GraphicsModActionData::Input::Projection&,
+                               GraphicsModActionData::Output::Projection*)
```

Comment:
And this action doesn't write into the output matrix at all, effectively discarding the matrix!

You probably want to rethink this.